src/igvfd/schemas/changelogs/tabular_file.md

```diff
@@ -2,7 +2,10 @@
 
 ### Minor changes since schema version 13
 
-* Extend `content_type` enum list to include `unfiltered global differential expression`, `unfiltered local differential expression` and `cell hashing barcodes`.
+* Extend `content_type` enum list to include `reporter experiment barcode`.
+* Extend `content_type` enum list to include `unfiltered global differential expression`.
+* Extend `content_type` enum list to include `unfiltered local differential expression`.
+* Extend `content_type` enum list to include `cell hashing barcodes`.
```

Comment:
Probably needs rebase since I think Shengcheng also fixed this separately

Reply:
fixed